documentation/book/assembly-custom-resources.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// getting-started.adoc
+
+[id='assembly-custom-resources-{context}']
+= Custom resources
```

Comment:
Insert a blank line after the heading. This appears to be preventing the first paragraph from rendering in my Asciidoctor browser extension.